crates/viewer/re_viewer/src/app_state.rs

```diff
@@ -69,6 +72,12 @@ pub struct AppState {
     #[serde(skip)]
     pub(crate) share_modal: crate::ui::ShareModal,
 
+    #[serde(skip)]
+    view_query_cache: HashMap<ViewId, ViewQueryCacheEntry>,
+
+    #[serde(skip)]
+    visualizable_entities_cache: HashMap<ViewId, VisualizableEntitiesCacheEntry>,
```

Comment:
need to find a better place for this. Probably best to refactor this together with `ViewState`, but that exceeds the scope of the PR a bit. So for starters I think these should go into a shared datastructure that lives on a cache struct that is `impl Cache` (and lazily created with other caches) so we can also easily oversee its memory usage